client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -94,6 +101,10 @@ function getJetpackPluginUrl( slug ) {
 	return `/plugins/jetpack/${ slug }`;
 }
 
+function getPluginDetailUrl( slug, siteSlug ) {
+	return `/plugins/${ slug }/${ siteSlug }`;
+}
```

Comment:
looks like this is only used once and isn't really helping much, maybe we should consider inlining it?